### PR TITLE
Update CI script for new release model

### PR DESCRIPTION
**Pull request by qinezh**
Set up a new CI to make *stable* branch sync with *dev* branch in real time automatically in the *first two* weeks.

### CODE REVIEW

File: tools/Deployment/util.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+      sp.stderr.on("data", function(data) {
+        logger.warn(data.toString());
+      });
+      sp.on("close", function(code) {
+        if (code === 0) {
+          logger.info("Finishing command: " + command + " " + argStr);
+          process.chdir(currentDir);
+          resolve();
+        } else {
+          reject(new Error("Error occurs while running " + command + " " + argStr + ", Exited with code: " + code ));
```
Comment (hellosnow) [inline tools/Deployment/util.js]: >Exited [](start = 87, length = 6)

minor: existed

Comment (qinezh) [inline tools/Deployment/util.js]: exist? I mean exit code...

---
In reply to: [93368780](https://github.com/dotnet/docfx/pull/1021#discussion_r93368780) [](ancestors = 93368780)

**Comment by hellosnow** [comment]
:shipit:

File: tools/Deployment/util.js (hunk 1 above)
Comment (hellosnow) [inline tools/Deployment/util.js]: I mean should not upper case...

---
In reply to: [93369267](https://github.com/dotnet/docfx/pull/1021#discussion_r93369267) [](ancestors = 93369267,93368780)

**Pull request merged**